modmesh/app/euler1d.py

```diff
@@ -104,16 +104,16 @@ def update(self, adata, ndata):
         self.num.figure.canvas.draw()
 
 
-class SolverConfig():
+class GUIConfig:
     """
-    Configuration class for the solver.
+    Configuration class for the GUI.
 
-    This class provides a configuration interface for the solver, allowing
+    This class provides a configuration interface for the GUI, allowing
     users to set and retrieve parameters related to the simulation.
 
     Attributes:
```

Comment:
Please use Sphinx format for the attributes in the docstring.

Reply:
I have fixed it in https://github.com/solvcon/modmesh/pull/414/commits/d25d5c2e7a37da1a74cd142ebc39ee0bf43b76bd.